<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheAudioDbOptionsPage</class>
 <widget class="QWidget" name="TheAudioDbOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>364</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TheAudioDB cover art</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body&gt;
&lt;p&gt;This plugin loads cover art from &lt;a href=&quot;https://www.theaudiodb.com&quot;&gt;TheAudioDB&lt;/a&gt;. TheAudioDB is a community Database of audio artwork and data. Their content is only possible thanks to the hard work of volunteer editors. If you like this plugin, please consider &lt;a href=&quot;https://www.theaudiodb.com/forum-new/ucp.php?mode=register&quot;&gt;registering&lt;/a&gt; as an editor or supporting the TheAudioDB &lt;a href=&quot;https://www.patreon.com/thedatadb&quot;&gt;patreon campaign&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>Medium images</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="theaudiodb_cdart_use_always">
        <property name="text">
         <string>Always load medium images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="theaudiodb_cdart_use_if_no_albumcover">
        <property name="text">
         <string>Load only if no front cover is available</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="theaudiodb_cdart_use_never">
        <property name="text">
         <string>Never load medium images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="theaudiodb_use_high_quality">
     <property name="text">
      <string>Use high resolution images, if available</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
